app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Welcome <%= @email %>!</p>
+
+<p>You can confirm your account email through the link below:</p>
+
+<p><%= link_to 'Confirm my account', confirmation_url(@resource, confirmation_token: @token) %></p>
```

Comment:
- [ ] Great job so far 👏🏻 However the confirmation seems not to work because I don't get in my terminate any link to confirm my account after creating one. Here is my terminal screenshot:

Reply:
![image](https://user-images.githubusercontent.com/55507343/206369035-0436b118-b2df-4e92-899c-3462e93e5e15.png)
@BenMukebo  As the `ActionView` Error at the end of your screenshot said.
You have to provide the host to the `config.action_mailer.default_url_options`.
Which if you take a look at the screenshot provided is an Environment variable.

Also, if you notice in `config/environment/development.rb` as I show in the screenshot. You'll see that the delivery_method I'm using is :letter_opener a gem to show the email in the browser once it is sent.

Reply:
Hey @wandergithub !!